master/getting-started/mesos/tutorials/launching-tasks.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Launching Tasks
+---
+
+The following information describes how to launch Calico networked tasks in Mesos
+using Marathon.
+
+### Unified Containerizer
+
+Launch a Unified Containerizer task onto a Calico network by setting
+ `networkName` to the name of your Calico network. Its value should match the `"name"`
+ field of the `calico.conf` you configured when [Installing Calico for Mesos](../installation/integration)
+
+```json
+{
+  "id": "unified-task",
+  "cmd": "/usr/sbin/ip addr show && sleep 1000",
+  "ipAddress": {
+      "networkName": "calico"
+  }
+}
+```
+
+>Be sure to replace `/usr/sbin/ip` with the correct path to your IP binary if you are following along.
+
+The task's stdout output should show a Calico IP from the default Calico pool of `192.168.0.0/16`.
+
+### Docker Containerizer
+
+Calico for Docker uses a Profile based network security model which has
```

Comment:
Maybe say this approach uses Calicos Docker networking plugin to provide Calico networking and policy.  More examples of this can be seen in the Docker tutorials.